hist/histpainter/src/TGraphPainter.cxx

```diff
@@ -3144,8 +3175,8 @@ void TGraphPainter::PaintGraphBentErrors(TGraph *theGraph, Option_t *option)
 
    const Int_t kBASEMARKER=8;
    Double_t s2x, s2y, symbolsize, sbase;
-   Double_t x, y, xl1, xl2, xr1, xr2, yup1, yup2, ylow1, ylow2, tx, ty;
-   Double_t bxl, bxh, byl, byh;
+   Double_t x, y, exl, exh, eyl, eyh, xl1, xl2, xr1, xr2, yup1, yup2, ylow1, ylow2, tx, ty;
```

Comment:
Why declaring the variables here ? It would be clearer for code readability to declare them where they are used

Reply:
done